sbpy/spectroscopy/tests/test_spec_remote.py

```diff
@@ -0,0 +1,200 @@
+import astropy.units as u
+from .. import Spectrum, einstein_coeff
+from astropy.tests.helper import remote_data
+from astropy.table import Table
+import numpy as np
+from astroquery.lamda import Lamda
+from astroquery.jplspec import JPLSpec
+from sbpy.activity.gas import Haser, photo_timescale
+
+import os
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'data')
+    return os.path.join(data_dir, filename)
+
+
+@remote_data
+def test_remote_prodrate_simple_hcn():
+
+    hcn = Table.read(data_path('HCN.csv'), format="ascii.csv")
```

Comment:
Use blank lines to separate logical groups in the code, many of them could be removed.